handler/actions.go

```diff
@@ -466,77 +476,81 @@ func (h *Handler) kick(ea *EventAction) error {
 		return err
 	}
 
-	matches := h.getMatches(ea.Action, ev.Text)
-	if len(matches) != 1 {
-		h.reply(ea, msgMustSpecifyUser, true)
-		return nil
-	}
-	uToKick, err := h.getUser(matches[0])
-	if err != nil {
-		log.Errorf("%+v", err)
-		h.reply(ea, msgUknownUser, true)
-		return err
-	}
-
-	count := 0
-	for _, res := range h.data.GetResources() {
-		pos, err := h.data.GetPosition(uToKick, res.Name, res.Env)
+	if (len(h.admins) == 0) || (util.InSlice(h.admins, u.Name)) {
```

Comment:
Probably worth making a small helper function for this since you're doing it in a couple of places

```go
// HasAdminAccess returns if the specified user has access to admin features. If no admins are defined
// at runtime, all users will have admin access 
func (h *Handler) HasAdminAccess(user string) bool {
	return len(h.admins) == 0 || util.InSlice(h.admins, user)
}
```

Reply:
Could you do an early return here instead of the big `if/else`?

```go
if !h.HasAdminAccess(u.Name) {
    h.reply(ea, "Error, your user is not authorized to run the command `kick`.", false)
    return nil
}

Reply:
So obvious that this is how it should've been done, thank you! Done!